Doc/library/threading.rst

```diff
@@ -71,6 +71,13 @@ This module defines the following functions:
 
    .. versionadded:: 3.8
 
+.. data:: __excepthook__
+
+   Holds the original value of :func:`excepthook`. It is saved so that the
```

Comment:
Since the sys module also has the same name, IMO it would help to explicitly mention the module name :-)

```suggestion
   Holds the original value of :func:`threading.excepthook`. It is saved so that the
```